examples/heating_and_cooling/src/run_case.py

```diff
@@ -142,7 +151,45 @@ def constraints(self, ensemble_member):
         # that a pipe can absorb heat
         # heat_to_discharge_test(heat_problem, results)
 
-
+        # heat_demands_assets = heat_problem.energy_system_components.get("heat_demand", [])
+        # heat_demands_result = {key:[] for key in heat_demands_assets}
+        # for demand in heat_demands_assets:
+        #     demand_result_values = heat_problem.get_timeseries(
+        #             f"{demand}.target_heat_demand", 0
+        #         ).values
+        #     heat_demands_result[demand] = demand_result_values
```

Comment:
Yes, there are a few comment blocks around I was just keeping in case I broke something with my own code. I'll delete them all before the final commit for review.